content/en/docs/05/labs/springboot-example.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: example-spring-boot
+  name: example-spring-boot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-spring-boot
+  template:
+    metadata:
+      labels:
+        app: example-spring-boot
+    spec:
+      containers:
+      - image: quay.io/acend/example-spring-boot # FIXME
```

Comment:
the correct URL would be: quay.io/acend/prometheus-training-spring-boot-example:latest 
But I assume we'll have to mirror this one since the customer might not have internetaccess?